packages/block-editor/src/components/text-decoration-control/README.md

```diff
@@ -0,0 +1,39 @@
+# TextDecorationControl
+
+![TextDecorationControl Element in Inspector Control](https://raw.githubusercontent.com/WordPress/gutenberg/HEAD/docs/assets/text-decoration-component.png?raw=true)
+
+## Table of Contents
+
+1. [Usage](#usage)
+3. [Properties](#props)
+
+## Usage
```

Comment:
It might be best if we still with a single code block for the usage example to be consistent with other readmes, such as the one you added for `TextTransformControl`.

P.S. That readme also has the same issues with its import statement and it being an experimental component.